jazz_test-lambda/index.js

```diff
@@ -0,0 +1,118 @@
+const responseObj = require("./components/response.js"); //Import the response module.
+const configObj = require("./components/config.js"); //Import the environment data.
+const logger = require("./components/logger.js"); //Import the logging module.
+const utils = require("./components/utils.js");
+const aws = require('aws-sdk');
+const validateARN = utils.validateARN;
+const execStatus = utils.execStatus();
+var handler = (event, context, cb) => {
+  'use strict';
+  //Initializations
+  var errorHandler = errorHandlerModule();
+  var config = configObj(event);
+  logger.init(event, context);
+  var AWS_REGION;
+  try {
+    var testResponse = {
+      "execStatus": null,
+      "payload": null,
+    };
+    if (!event && !event.method && event.method !== 'POST') {
+      return cb(JSON.stringify(errorHandler.throwNotFoundError("Method not found")));
+    }
+    if (!event.body) {
+      return cb(JSON.stringify(errorHandler.throwInputValidationError("Event Body not Defined")));
+    }
+    if (!validateARN(event.body.functionARN)) {
+      return cb(JSON.stringify(errorHandler.throwInputValidationError("Function ARN is invalid")));
+    }
+    if (!event.body.inputJSON) {
+      return cb(JSON.stringify(errorHandler.throwInputValidationError("Input for function is not defined")));
+    }
+    var functionARN = event.body.functionARN;
+    var arnvalues = functionARN.split(":");
+    AWS_REGION = arnvalues[3]; //["arn","aws","lambda","us-east-1","000000""] spliting FunctionARN to get the aws-region 
+    var inputJSON = event.body.inputJSON;
+    invokeLambda(functionARN, inputJSON, AWS_REGION).then((data) => {
+      if (data && data.StatusCode === 200) {
+        testResponse.payload = data;
+        if (!data.FunctionError) {
+          //Function Executed Succesfully Without Error 
+          testResponse.execStatus = execStatus.success;
+        } else {
+          if (data.FunctionError === "Handled") {
+            testResponse.execStatus = execStatus.handledError;
+          } else if (data.FunctionError === "Unhandled") {
+            // Function Execution Had Unhandled Error 
+            testResponse.execStatus = execStatus.unhandledError;
+          }
+        }
+      } else {
+        // Function Falied |Cause Unknown|TEST FAILED 
+        return cb(JSON.stringify(errorHandler.throwInternalServerError("Unknown error occured")));
+      }
+      testResponse.payload = data;
+      return cb(null, responseObj(testResponse, event.body));
+    }).catch((err) => {
+      // Funtion Failed To Be Invoked |TEST FAILED
+      testResponse.execStatus = execStatus.functionInvocationError;
+      testResponse.payload = err;
+      return cb(null, responseObj(testResponse, event.body));
+    });
+  } catch (err) {
+    return cb(JSON.stringify(errorHandler.throwInternalServerError("Failed to invoke lambda")));
+  }
+};
+
+var invokeLambda = (functionARN, inputJSON, AWS_REGION) => {
+  'use strict';

```

Comment:
why?

Reply:
jenkins job gave an error when putting strict in the global level, with the suggestion of using it at the functional level